python/tk_desktop/rpc.py

```diff
@@ -280,10 +287,24 @@ def run(self):
                     ready = False
             else:
                 # can use select on osx and linux
-                (rd, _, _) = select.select(
-                    [self.server._listener._socket], [], [], self.LISTEN_TIMEOUT
-                )
-                ready = len(rd) > 0
+                try:
+                    (rd, _, _) = select.select(
+                        [self.server._listener._socket], [], [], self.LISTEN_TIMEOUT
+                    )
+                    ready = len(rd) > 0
```

Comment:
Is it necessary the comparison `len(rd) > 0` ? Because I think that it is possible to use just `len(rd)`

Reply:
It seems `ready` was defined as boolean type so this is to keep type consistency.